app.js

```diff
@@ -0,0 +1,18 @@
+const openBtn = document.querySelector('.hamburger');
+const closeBtn = document.querySelector('.close-btn');
+const mobileMenu = document.querySelector('.mobile-menu');
+const navLinks = document.querySelectorAll('.menu-links');
+
+openBtn.addEventListener('click', function (){
+  mobileMenu.classList.add('show-menu');
+});
+
+closeBtn.addEventListener('click', function (){
+  mobileMenu.classList.remove('show-menu');
+});
+
+navLinks.forEach(function(link) {
+  link.addEventListener('click', function(){
+    mobileMenu.classList.remove('show-menu');
+  });
+}) ;
```

Comment:
- [ ] Consider adding a filter to the layer under the mobile menu to match the design 

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/195078561-d1b516a8-b663-42fd-b6f9-4b744f7dca72.png)| ![image](https://user-images.githubusercontent.com/53402600/195078634-9b1191f3-f4cd-4b57-b87a-8977b16f36e7.png) |